engine/src/lextable.cpp

```diff
@@ -2458,4 +2449,11 @@ const uint2 table_sizes[] =
     ELEMENTS(visual_table),
 	ELEMENTS(server_table),
 };
-extern const uint4 table_sizes_size = ELEMENTS(table_sizes);
+const static uint4 table_sizes_size = ELEMENTS(table_sizes);
+
+/* All keyword tables are const static in this translation unit, but we allow
+ * access to them through well-defined global variables. */
+LT **MCkeywordtablepointers = (LT **)table_pointers;
+uint4 MCkeywordtablepointerssize = table_pointers_size;
+uint2 *MCkeywordtablesizes = (uint2 *)table_sizes;
+uint4 MCkeywordtablesizessize = table_sizes_size;
```

Comment:
Could the global variables be defined as `const` instead of the const-ness being cast away? e.g `const LT * const * const MCkeywordtablepointers = table_pointers;` (a constant pointer to constant pointers to constant LTs)
If not perhaps `const_cast` should be used here to signify what is being done.